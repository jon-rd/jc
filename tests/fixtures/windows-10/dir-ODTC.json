[{"date":"12/07/2019","time":"02:09 AM","dir":false,"size":54784,"filename":"hmmapi.dll","parent":"C:\\Program Files\\Internet Explorer","epoch":1575713340},{"date":"12/07/2019","time":"02:09 AM","dir":false,"size":224768,"filename":"ielowutil.exe","parent":"C:\\Program Files\\Internet Explorer","epoch":1575713340},{"date":"12/07/2019","time":"02:09 AM","dir":false,"size":54784,"filename":"ExtExport.exe","parent":"C:\\Program Files\\Internet Explorer","epoch":1575713340},{"date":"12/07/2019","time":"02:09 AM","dir":false,"size":515072,"filename":"iediagcmd.exe","parent":"C:\\Program Files\\Internet Explorer","epoch":1575713340},{"date":"12/07/2019","time":"02:09 AM","dir":false,"size":421888,"filename":"IEShims.dll","parent":"C:\\Program Files\\Internet Explorer","epoch":1575713340},{"date":"12/07/2019","time":"02:09 AM","dir":false,"size":504832,"filename":"ieinstal.exe","parent":"C:\\Program Files\\Internet Explorer","epoch":1575713340},{"date":"12/07/2019","time":"02:09 AM","dir":false,"size":48536,"filename":"sqmapi.dll","parent":"C:\\Program Files\\Internet Explorer","epoch":1575713340},{"date":"12/07/2019","time":"02:14 AM","dir":true,"size":null,"filename":"images","parent":"C:\\Program Files\\Internet Explorer","epoch":1575713640},{"date":"12/07/2019","time":"02:14 AM","dir":true,"size":null,"filename":"..","parent":"C:\\Program Files\\Internet Explorer","epoch":1575713640},{"date":"12/07/2019","time":"02:14 AM","dir":true,"size":null,"filename":"SIGNUP","parent":"C:\\Program Files\\Internet Explorer","epoch":1575713640},{"date":"12/07/2019","time":"02:14 AM","dir":true,"size":null,"filename":".","parent":"C:\\Program Files\\Internet Explorer","epoch":1575713640},{"date":"12/07/2019","time":"02:49 AM","dir":true,"size":null,"filename":"en-US","parent":"C:\\Program Files\\Internet Explorer","epoch":1575715740},{"date":"12/07/2019","time":"02:49 AM","dir":false,"size":819136,"filename":"iexplore.exe","parent":"C:\\Program Files\\Internet Explorer","epoch":1575715740},{"date":"03/24/2021","time":"03:15 PM","dir":false,"size":0,"filename":"file name.txt","parent":"C:\\Program Files\\Internet Explorer","epoch":1616624100}]
