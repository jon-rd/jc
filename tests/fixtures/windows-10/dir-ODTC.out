 Volume in drive C has no label.
 Volume Serial Number is 246B-3B5A

 Directory of C:\Program Files\Internet Explorer

12/07/2019  02:09 AM            54,784 hmmapi.dll
12/07/2019  02:09 AM           224,768 ielowutil.exe
12/07/2019  02:09 AM            54,784 ExtExport.exe
12/07/2019  02:09 AM           515,072 iediagcmd.exe
12/07/2019  02:09 AM           421,888 IEShims.dll
12/07/2019  02:09 AM           504,832 ieinstal.exe
12/07/2019  02:09 AM            48,536 sqmapi.dll
12/07/2019  02:14 AM    <DIR>          images
12/07/2019  02:14 AM    <DIR>          ..
12/07/2019  02:14 AM    <DIR>          SIGNUP
12/07/2019  02:14 AM    <DIR>          .
12/07/2019  02:49 AM    <DIR>          en-US
12/07/2019  02:49 AM           819,136 iexplore.exe
03/24/2021  03:15 PM                 0 file name.txt
               9 File(s)      2,643,800 bytes
               5 Dir(s)  35,231,817,728 bytes free
